app/views/shared/article/_attachments_display.html.erb

```diff
@@ -1,14 +1,41 @@
-<% 5.times do |n| %>
-  <%= render partial: "shared/article/file_display", locals: {
-    file_field:           "file_#{n + 1}".to_sym,
-    file_language_field:  "file_#{n + 1}_language".to_sym,
-    file_label_field:     "file_#{n + 1}_label".to_sym
-  } %>
-<% end %>
+<% if cms_block_content(:file_1).try(:file)&.present? ||
```

Comment:
Hmmm, what I would do is something like this:

```
<% array = [] %>
<% 5.times do |n| %>
  <% array << cms_block_content("file_#{n + 1}".to_sym).try(:file).present? %>
<% end %>
```
Then you can check if there are any true values in that array
```
<% if array.include? true %>
```

But I think would need shifting to a helper method, because the end result is just to be able to do something like...
```
<% if any_files_for?(5) %>
```
(methods that return a boolean in ruby are usually named with a question mark at the end)